.setup/bin/sample_courses/models/course/course_generate_utils.py

```diff
@@ -104,21 +111,29 @@ def make_course_json(self) -> None:
                 max_auto = 0
                 max_ta = 0
 
-                print_grades = True if g_type != 0 or (gradeable.submission_open_date < NOW) else False
-                release_grades = (gradeable.has_release_date is True) and (gradeable.grade_released_date < NOW)
+                if g_type != 0 or (gradeable.submission_open_date < NOW):
+                    print_grades = True
+                else:
+                    print_grades = False
```

Comment:
You could just do something like this to stay under the character limit I believe:
```suggestion
                print_grades = g_type != 0 or gradeable.submission_open_date < NOW
```